src/services/mastodon/service.json

```diff
@@ -1,5 +1,5 @@
 {
-  "id": "mastodon",
-  "name": "Mastodon",
-  "url": ""
-}
+    "id": "mastodon",
+    "name": "Mastodon",
+    "url": "https://mastodon.social/about"
```

Comment:
mastodon can be hosted anywhere, please revert this change

Reply:
ah ok mastodon.social is the main instance, makes sense since users can change the URL anyway

Reply:
ok I will fix it